config/config.example.yml

```diff
@@ -183,3 +183,58 @@ sync_permissions:
       - "process_job_status : read -> job_status : read"
       # different permission (match), otherwise all jobs/outputs become available.
       - "process_job_status : read -> process_description : read-match"
+stac_permissions:
+    services:
+      api:
+        stac_collection:
+          - name: stac
+            type: service
+          - name: stac
+            type: route
+          - name: collections
+            type: route
+          - type: route
+            field: resource_display_name # Use the resource_display_name for permission mapping.
+            regex: '[\w]+:[\w\/]+' # This will extract the display name (ex: thredds:birdhouse/testdata/xclim/cmip6).
+        stac_item:
+          - name: stac
+            type: service
+          - name: stac
+            type: route
+          - name: collections
+            type: route
+          - name: "{collectionId}"
+            type: route
+          - name: items
+            type: route
+          - type: route
+            field: resource_display_name # Use the resource_display_name for permission mapping.
+            regex: '[\w]+:[\w\/.-]+' # This will extract the display name (ex: thredds:birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc)
+
+      thredds: 
+        thredd_collection:
+          - name: thredds
+            type: service            
+          - type: directory
+            regex: '(?<=:)[\w\/]+' # Match everything after ":" but before last "/" (ex: thredds:birdhouse/testdata/xclim/cmip6 will match to : birdhouse/testdata/xclim/cmip6)
```

Comment:
Not sure about the `but before last "/"` part here.
I think with the current regex it would match anything before any character that is not `\w` or `/`. 

It might just be a copy paste from below description which needs to be updated.
Note that if the path is `thredds:birdhouse/testdata/test.txt` for example, it might match `birdhouse/testdata/test` here.

Reply:
you right it should be `(?<=:).*\/?(?=\/)`